components/README.md

```diff
@@ -0,0 +1,141 @@
+</details>
+
+### Standard
+
+This component gives the user option to confirm the information.
+
+<img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusDialogSimplePreview,1,light].png" alt="alt text" style="zoom: 80%;" />
+
+
+
+#### Advanced
+
+This component gives  the user the choice out of two options, usually dismiss or confirmation.
+
+<img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusDialogAdvancedPreview,1,light].png" alt="alt text" style="zoom: 80%;" />
+
+
+
+## Variants
+
+#### Error
+
+<details>
+    <summary>Code snippet</summary>
+    <br>
+https://github.com/appunite/Loudius/blob/8e616a9a05e668d11425a40c9f452eb3091dcc2a/components/src/main/java/com/appunite/loudius/components/components/LoudiusErrorDialog.kt#L29-L47
+</details>
+
+<img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusErrorDialogPreview,1,light].png" style="zoom:80%;" />
+
+# Screens
+
+Components replacing the whole screen, for example when indicating a error.
+
+#### Error
+
+Component replacing the whole screen when indicating the error, with a custom action button and title.
+
+<details>
+    <summary>Code snippet</summary>
+    <br>
+https://github.com/appunite/Loudius/blob/8e616a9a05e668d11425a40c9f452eb3091dcc2a/components/src/main/java/com/appunite/loudius/components/components/LoudiusFullScreenError.kt#L39-L84
+</details>
+
+
+
+<img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusErrorScreenCustomTextsPreview,1,light].png" />
+
+# List item
+
+Component being displayed inside a list.
+
+## Variants
+
+#### Content with action
+
+Component allows to display the "action" on the right of the component.
+
+<details>
+    <summary>Code snippet</summary>
+    <br>
+https://github.com/appunite/Loudius/blob/8e616a9a05e668d11425a40c9f452eb3091dcc2a/components/src/main/java/com/appunite/loudius/components/components/LoudiusOutlinedButton.kt#L37-L61
+</details>
+
+
+
+<img src="src\test\snapshots\images\com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusListItemContentAndActionPreview,1,light].png" alt="com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusListItemContentAndActionPreview,1,light]" />
+
+
+
+#### Content with icon
+
+Component also takes a icon which is displayed on the left of the "content".
+
+
+
+<img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusListItemContentAndIconPreview,1,light].png" />
+
+
+
+#### Content only
+
+Component taking only the "content".
+
+
+
+<img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusListItemJustContentPreview,1,light].png" />
+
+# Buttons
```

Comment:
Question: What do you think about cut whole screen only to button?

Reply:
Ok, I read the description again and now I see why we have whole screens ;)

Reply:
yea exactly I do not think there is a way around it ;s

Reply:
I tried some approaches but in the end paparrazi will take a screenshot of the device so we will end up with abig screenshot, if you have some ideas let me know

Reply:
I was googling it for a while and unfortunately I couldn't find any helpful information :(

Reply:
I am going to leave as it is for now. Maybe something for future improvements